### ISSUE TITLE
licensezero offer does not write a licensezero.json?

### ISSUE THREAD

**Issue by RobRoseKnows** [licensezero/cli#43]
Is the licensezero.json format retired? I created a new project, added the Prosperity license and then created an offer but it didn't write a licensezero.json to the folder. It'll eventually be a go project, but I haven't written any go code yet, do I need a `go.mod` file first?

**Comment by kemitchell** [comment]
@RobRoseKnows, indeed, the `licensezero.json` metadata file is retired, as is the CLI for users, at least for now. Those decisions were taken thanks to feedback strongly encouraging overall simplification of License Zero as a system. That said, developers are taking advantage of various other ways to announce their offers, in documentation and package metadata.

A `go.mod` file defines a module in Go's recently adopted official module system. You will almost certainly want one. But at least as far as I know, `go.mod` isn't a place for licensing other general project metadata.
